src/api/warp.ts

```diff
@@ -152,10 +155,29 @@ function createQueryParamHash(evalOptions: Partial<EvaluationOptions>): string {
   return hash.digest('hex');
 }
 
+function createSortKeyBlockHeightHash({
+  sortKey,
+  blockHeight,
+}: {
+  sortKey: string | undefined;
+  blockHeight: number | undefined;
+}) {
+  const hash = createHash('sha256');
```

Comment:
What's the motivation for using a hash of these inputs for the cache key rather than literal strings? What are the tradeoffs? e.g. does one provide better cache key collision protection?

Reply:
eh, probably a bit of overkill but primarily to create fixed-sized/length strings for the cache keys. given these are user-provided inputs they have high cardinality and can also be `undefined`.

```
sortKey: undefined
blockHeight: undefined
resulting hash: `465039ec2625fb7eace80f9a3e9bd63dd04b0c55d971f6af1894f4b8c1b3c0ad`
```
vs. 
`undefined-undefined` or adding logic to those that result in an empty string.

Reply:
That essentially confirms my assumptions. Part of me wonders if the hashing layer is worth it though given that:
1) Under the hood, PromiseCache, which is built on the simple-cache library's EphemeralCache, has at its base at Map object which is benefiting from all the niceties getting the performance tradeoffs you want from customized hash tables. So, at a base level of running the cache optimally, we're doing redundant work and it may actually be hampering some of what the base implementation is trying to optimize for.
2) We may be making debugging harder for ourselves if hash keys we want to inspect are then hashed again, losing some of their original context during the process of the redundant hashing.

I think what you have is ok to ship, but this might be something you want to revisit later.